hrjobcontract/scss/hrjc/partials/_forms.scss

```diff
@@ -124,3 +122,37 @@ $customselect-padding-right: $customselect-caret-width + 12;
         vertical-align: top;
     }
 }
+
+.form-control {
+  @include box-shadow($crm-form-control-shadow);
+
+  &:focus {
+    @include box-shadow($crm-form-control-shadow);
+    border-color: $brand-primary;
+  }
+
+  .input-group-addon,
+  .input-group-btn,
+  .input-group .form-control {
+    display: table-cell;
```

Comment:
this stuff is for layouts, we are supposed to change only colors in this PR

Reply:
This changes are made to keep input boxes same as shoreditch